ccm_web/server/src/canvas/canvas.service.ts

```diff
@@ -29,9 +29,7 @@ const requestorOptions: GotOptions = {
   retry: {
     limit: 2,
     methods: ['POST', 'GET', 'PUT', 'DELETE'],
-    // TODO: After got@12 upgrade, replace following list with something like…
-    // got.defaultInternals.retry.statusCodes.concat(403)
-    statusCodes: [403, 408, 413, 429, 500, 502, 503, 504, 521, 522, 524]
+    statusCodes: got.defaults.options.retry.statusCodes.concat([403])
```

Comment:
This Looks Good. As the way it is implemented is it is concatenating to existing list.

Reply:
Yes, that's a literal reading of the code.